package.json

```diff
@@ -133,14 +133,15 @@
     "yargs": "^17.1.1"
   },
   "devDependencies": {
-    "@wdio/cli": "^7.16.16",
-    "@wdio/local-runner": "^7.12.4",
-    "@wdio/mocha-framework": "^7.12.4",
-    "@wdio/sauce-service": "^7.16.16",
-    "@wdio/selenium-standalone-service": "^7.12.5",
-    "@wdio/shared-store-service": "^7.16.16",
-    "@wdio/spec-reporter": "^7.20.8",
-    "@wdio/static-server-service": "^7.10.1",
+    "@types/fs-extra": "^11.0.1",
+    "@wdio/cli": "^8.3.3",
+    "@wdio/local-runner": "^8.3.3",
+    "@wdio/mocha-framework": "^8.3.0",
+    "@wdio/sauce-service": "^8.3.3",
+    "@wdio/selenium-standalone-service": "^8.3.2",
+    "@wdio/shared-store-service": "^8.3.5",
```

Comment:
@wdio/shared-store-service should be in dependencies, not devDependencies since this is used in index.ts

Reply:
I have moved it to dependencies now. Thanks for the suggestion :)